hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestS3BucketManager.java

```diff
@@ -60,24 +60,6 @@ public void init() throws IOException {
     bucketManager = new BucketManagerImpl(metaMgr);
   }
 
-  @Test
-  public void testCreateS3Bucket() throws IOException {
```

Comment:
Why is this test not needed any more?

Reply:
Because S3BucketCreateRequest is tested through TestS3BucketCreateRequest. And create logic in S3BucketManagerImpl is no longer used.